freqtrade/data/dataprovider.py

```diff
@@ -44,36 +44,50 @@ def available_pairs(self) -> List[Tuple[str, str]]:
 
     def ohlcv(self, pair: str, ticker_interval: str = None, copy: bool = True) -> DataFrame:
         """
-        get ohlcv data for the given pair as DataFrame
-        Please check `available_pairs` to verify which pairs are currently cached.
+        Get ohlcv data for the given pair as DataFrame
+        Please check `self.available_pairs` to verify which pairs are currently cached.
```

Comment:
`self.available_pairs` is not necessarily correct, assuming it's called from the strategy... (and i assume doc-strings are mostly for callers not located in this class, but in 
a) a strategy
b) a interactive environment

Reply:
Docstring improved, done, pls verify.